eZ/Publish/Core/Persistence/Legacy/Content/Type/ContentUpdater/Action/RemoveField.php

```diff
@@ -94,6 +94,13 @@ function ($versionNo) use ($contentId) {
             );
         }
 
+        // Delete from relations storage
+        if (
+            $this->fieldDefinition->fieldType === 'ezobjectrelationlist'
```

Comment:
We should not hardcode this here, can we take it from some constant? FieldType perhaps?